modules/lambdas/variables.tf

```diff
@@ -118,10 +118,6 @@ variable "orca_recovery_retry_interval" {
   description = "Number of seconds to wait between recovery failure retries."
 }
 
-variable "orca_recovery_retry_backoff" {
```

Comment:
Did you mean to delete this? I think this is still used.

Reply:
nope my bad